src/main/webapp/WEB-INF/tags/admin/search/instructorRow.tag

```diff
@@ -60,14 +60,14 @@
             <%-- Email --%>
             <li class="list-group-item list-group-item-success has-success">
                 <strong>Email</strong>
-                <input value="${instructor.email}" readonly="readonly" class="form-control" />
+                <input value="${instructor.email}" readonly class="form-control" />
```

Comment:
Since you've touched this line, change that closing `/>` to simply `>` as per non-XHTML standard. Apply to all other lines in this PR as well.